<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Province:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Postal Code: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> City:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Forecast:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProvince">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Eg: ON or Ontario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCity">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Eg: Sudbury or sudbury</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPostalCode">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>90</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Eg: P8C 2Q1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn1Day">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>1 Day</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWeather">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Weather Forecast</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Powered by Weatherbit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>1151</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Layout">
     <item>
      <widget class="QFrame" name="day1">
       <layout class="QVBoxLayout" name="Day1">
        <item>
         <widget class="QLabel" name="Date1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Icon1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="temper1">
          <item>
           <widget class="QLabel" name="Temp1">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hilow">
            <item>
             <widget class="QLabel" name="Maxtemp1">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="MinTemp1">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="day2">
       <layout class="QVBoxLayout" name="Day2">
        <item>
         <widget class="QLabel" name="Date2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Icon2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="temper2">
          <item>
           <widget class="QLabel" name="Temp2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hilow2">
            <item>
             <widget class="QLabel" name="Maxtemp2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="MinTemp2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="day3">
       <layout class="QVBoxLayout" name="Day3">
        <item>
         <widget class="QLabel" name="Date3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Icon3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="temper3">
          <item>
           <widget class="QLabel" name="Temp3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hilow3">
            <item>
             <widget class="QLabel" name="Maxtemp3">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="MinTemp3">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="day4">
       <layout class="QVBoxLayout" name="Day4">
        <item>
         <widget class="QLabel" name="Date4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Icon4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="temper4">
          <item>
           <widget class="QLabel" name="Temp4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hilow4">
            <item>
             <widget class="QLabel" name="Maxtemp4">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="MinTemp4">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="day5">
       <layout class="QVBoxLayout" name="Day5">
        <item>
         <widget class="QLabel" name="Date5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Icon5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="temper5">
          <item>
           <widget class="QLabel" name="Temp5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hilow5">
            <item>
             <widget class="QLabel" name="Maxtemp5">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="Mintemp522">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="day6">
       <layout class="QVBoxLayout" name="Day6">
        <item>
         <widget class="QLabel" name="Date6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Icon6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="temper">
          <item>
           <widget class="QLabel" name="Temp6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hilow6">
            <item>
             <widget class="QLabel" name="Maxtemp6">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="MinTemp6">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="day7">
       <layout class="QVBoxLayout" name="Day7">
        <item>
         <widget class="QLabel" name="Date7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Icon7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="temper7">
          <item>
           <widget class="QLabel" name="Temp7">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hilow7">
            <item>
             <widget class="QLabel" name="Maxtemp7">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="MinTemp7">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnCurrentLocation">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Use
Current Location</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn7Day">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>7 Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGreeting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Greeting</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>150</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showWeather">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>190</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Weather</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Saved Cities</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save City</string>
    </property>
   </widget>
   <widget class="QFrame" name="locationFrame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>821</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="lblLocationInfo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
